doc/expressionLanguage.md

```diff
@@ -175,3 +176,35 @@ The following table shows expressions and their expected outcomes for a measure
 | '@value * 5.2'              | 31.2                  |
 | '"Pruebas " + "De Strings"' | 'Pruebas De Strings'  |
 | '@name value is @value'     | 'DevId629 value is 6' |
+
+## <a name="ngsiv2"/> NGSIv2 support
+
+As it is explained in previous sections, expressions can have two return types: String or Number, being the former one the default. Whenever a expression is executed without error, its result will be cast to the configured type. 
+
+On one hand, in NGSIv1 since all attributes'values are string, in the expression parser the expression type is set always to String and the transformation of the information coming from the SouthBound is done using replace instruction. Therefore, values sent to the CB will always be Strings. This can be seen in (#execution) example.
+
+On the other hand, NGSIv2 fully supports all the types described in the JSON
+specification (string, number, boolean, object, array and null). Therefore, the result of an expression must be cast to the appropriate type in order to avoid inconsistencies between the type field for an attribute and the type of the value that is being sent.
+
+Currently, the expression parser does not support JSON Arrays and JSON document. A new issue has been created to address this aspect https://github.com/telefonicaid/iotagent-node-lib/issues/568. For the rest of types the workflow will be the following:
+
+1. Variables will be casted to Number or String depending on the expression type.
+2. The expression will be applied
+3. The output type will be casted again to the original attribute type.
+
+E.g.: if a device with the following provisioning information is provisioned in the IoTAgent:
+```
+{
+   "name":"status",
+   "type":"Boolean",
+   "expression": '${@status *  20}'
+}
+```
+and a measurement with the following values arrive to the IoTAgent:
+```
+status: true
+```
+
+The value true will be sent to the Context Broker since true will be converted to 1 before applying the expression and its reult (20) will be converted back to true (Everything With a "Value" is True in javacript).
```

Comment:
Typo: reult -> result

Reply:
Fixed in 776fe93a154bdd014f5bb7b37c8e165a52ab66d1